mkdocs.yml

```diff
@@ -0,0 +1,15 @@
+site_name: ddev Documentation
+edit_uri: edit/master/docs/
+copyright: DRUD Technology, LLC
+theme: readthedocs
```

Comment:
A quick search doesn't show much to chose from, https://github.com/mkdocs/mkdocs/wiki/MkDocs-Themes